laws/src/main/scala/cats/laws/AlternativeLaws.scala

```diff
@@ -0,0 +1,25 @@
+package cats.laws
+
+import cats.Alternative
+
+import cats.syntax.all._
+
+trait AlternativeLaws[F[_]] extends ApplicativeLaws[F] with MonoidKLaws[F] {
+  implicit override def F: Alternative[F]
+  implicit def algebra[A] = F.algebra[A]
+
+  def rightAbsorption[A, B](ff: F[A => B]): IsEq[F[B]] =
+    (F.empty[A] apply ff) <-> F.empty[B]
```

Comment:
Minor nitpick: could you prepend all laws with `alternative`? This way it is easier to see where these laws are coming from if `AlternativeLaws` is mixed into another law trait.